qt_ui/widgets/map/mapmodel.py

```diff
@@ -212,7 +212,7 @@ def showPackageDialog(self) -> None:
 
     @Property(str, notify=nameChanged)
     def name(self) -> str:
-        return self.tgo.name
+        return f"{self.tgo.name} ({self.tgo.control_point.name})"
```

Comment:
Add a new property for `description`? This isn't the name and getting the name right could matter for some uses.